Projects/Foundations/Schemas/Docs-Topics/I/Introduction/Introduction/introduction-007-references.json

```diff
@@ -35,8 +35,8 @@
                 },
                 {
                     "language": "EL",
-                    "text": "Κατεύθυνση",
-                    "updated": 1638977276823
+                    "text": "Direction (Κατεύθυνση)",
+                    "updated": 1641831264696
```

Comment:
You accidentally overwrite the Russian translation. Same problem in this File but absolutely fixable.